features/FEATURE_BLE/targets/TARGET_Cypress/TARGET_CYW43XXX/CyVscCmd.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 Cypress Semiconductor Corporation
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#include <mbed.h>
```

Comment:
This should not be here, but rather specific headers that are required here

Reply:
Done.